sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -721,8 +721,8 @@ object SQLConf {
         "shuffle partitions, but adaptively calculate the target size according to the default " +
         "parallelism of the Spark cluster. The calculated size is usually smaller than the " +
         "configured target size. This is to maximize the parallelism and avoid performance " +
-        "regression when enabling adaptive query execution. It's recommended to set this config " +
```

Comment:
@maryannxue is it really recommended?

Reply:
maybe just say `It's recommended to set this config to true on a busy cluster to male resource utilization more efficient (not many small tasks).`

Reply:
OK

Reply:
This suggestion contains a mistake right? It should be set to false in a busy cluster? https://github.com/apache/spark/pull/45437